src/screens/NotificationScreen/components/EventNotification/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import dayjs from 'dayjs';
+import { styles } from './style';
+import { dateFormat } from '../../../../components/constants';
+import { THEME_COLORS } from '../../../../components/theme';
+import { mockEventInfo, mockTeamInfo } from '../../../../shared/zustand/event/mockData';
+    
+const EventNotification = ({ event, eventType, onPress }) => {
+  return (
+    <TouchableOpacity style={styles.eventItem} onPress={() => {
+      const eventId = event['eventId'];
+      onPress(eventId);
+    }}>
+      <View style={styles.header}>
+        <View style={styles.infoPair}>
+          <Icon
+            name="calendar-clock"
+            size={18}
+            color={THEME_COLORS.BLACK}
+            style={styles.icon}
+          />
+
+          <Text numberOfLines={1} style={styles.title}>
+            {mockEventInfo[0].eventName}
+          </Text>
+        </View>
+        <View style={styles.infoPair}>
+          <Icon
+            name="check-bold"
+            size={18}
+            color={"#008000"}
```

Comment:
Try adding this color in `THEME_COLORS` and use it as an enum or specify the color in the `style.js`